discord/raw_models.py

```diff
@@ -357,6 +359,32 @@ def __init__(self, data: ThreadDeleteEvent) -> None:
         self.thread: Optional[Thread] = None
 
 
+class RawThreadMembersUpdate(_RawReprMixin):
+    """Represents the payload for a :func:`on_raw_thread_member_remove` event.
+
+    .. versionadded:: 2.0
+
+    Attributes
+    ----------
+    thread_id: :class:`int`
+        The ID of the thread that was updated.
+    guild_id: :class:`int`
+        The ID of the guild the thread is in.
+    member_count: :class:`int`
+        The number of members in the thread.
```

Comment:
Should probably mention this is just for UI purposes and caps out at 50, similar to the `Thread.member_count` docstring.

Reply:
I wasn't sure if you meant to say `this is just for UI purposes` to me or for the docstring. English isn't my main language.